src/webchat-ui/components/history/ChatScroller.tsx

```diff
@@ -62,6 +64,22 @@ export class ChatScroller extends React.Component<InnerProps, IState> {
         }
     }
 
+    scrollToPosition = (position: number) => {
+        const root = this.rootRef.current;
+
+        // we need the container reference to perform a scroll on it
+        if (!root)
+            return;
+
+        try {
+            root.scroll({
+                top: position - 70
+            });
+        } catch (e) {
+            root.scrollTop = position;
```

Comment:
why this error handling?

>     root.scrollTop = position;

Reply:
I just used the same idea from the existing method "scrollToBottom". If the instance method "scroll" fails we do scrolling by setting "scrollTo" instance parameter.
Anyway, if we merge the two methods as suggested in the above comment, this error handling will not change.